scout/server/links.py

```diff
@@ -782,6 +787,72 @@ def alamut_link(
     )
 
 
+def alamut_gene_link(
+    institute_obj,
+    gene_obj,
+    build=None,
+):
+    """Compose a link which open up variants in the Alamut software.
+    Alamut links require some settings from the institute object.
+    The link is rendered on the gene side as Alamut has issues with GATK style genome coordinates for indels.
+
+    We utilise the "canonical" transcript for each gene, as well as the HGVS c. string.
+
+    Args:
+        institute_obj(scout.models.Institute)
+        variant_obj(scout.models.Variant):
+        build(str): "37" or "38"
+
+    Returns:
+        url_template(str): link to Alamut browser
+    """
+
+    if current_app.config.get("HIDE_ALAMUT_LINK"):
+        return False
+
+    if not institute_obj:
+        return False
+
+    (alamut_key, search_verb, alamut_key_arg, alamut_inst_arg) = _get_alamut_config(institute_obj)
+
+    url_template = (
+        "http://localhost:10000/{search_verb}?{alamut_key_arg}{alamut_inst_arg}request={this[canonical_transcript]}{build_str}:"
+        "{hgvs_identifier}"
+    )
+
+    build = build or 37
+    build_str = ""
+    if build == 38:
+        build_str = "(GRCh38)"
+
+    return url_template.format(
+        search_verb=search_verb,
+        alamut_key_arg=alamut_key_arg,
+        alamut_inst_arg=alamut_inst_arg,
+        this=gene_obj,
+        build_str=build_str,
+        hgvs_identifier=quote(gene_obj["hgvs_identifier"]),
+    )
+
+
+def _get_alamut_config(institute_obj: dict) -> tuple[str, ...]:
```

Comment:
On my first attempt to set this up, I get errors from this line:

```
TypeError: 'type' object is not subscriptable
```

Which seems to be due to that this import does not import Tuple (and we seem to be using an older Python version):

```
from typing import List, Optional
```

Need to run now, but will continue testing later!

Reply:
Oh, what Python version are you running? That's good info. I think this is ok from 3.9 onwards, or maybe it was even 3.8 when they started allowing the builtins for hinting. Anyway, thanks, I'll update to the typing classes instead since you have issues with it!

Reply:
Yes, looks like we are running 3.8 at the moment! What version do you guys use for it?

Reply:
I also run 3.8 and I get these types or errors sometimes when PRs use more modern PEP standards. But we are supporting 3.8 until it gets deprecated so it should be **T**uple 👍🏻

Reply:
We run the tests on 3.11 to make sure we are tracking somewhat. Its even time to bump that to 3.12. But there is really a mixed bag of old versions in use. I happen to run my local on 3.9. There are even some installations with 3.7 going strong. All things docker is on 3.8 atm so that seems like a good place to be. As @northwestwitch says, we'll try to follow the python release schedule, meaning keeping 3.8 compatibility until 2024-10.

Reply:
Ok, updated. I think I'll call it a day with that. Cheers!

Reply:
OK good to know! We are running the docker version, thus version 3.8 (so I guess we'll update when you update the Docker container version).

Thanks, I'll test again today!